product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx

```diff
@@ -5,56 +5,71 @@ navTitle: "Prerequisites"
 
 Postres Enterprise Manager requires a Postgres server running either [EDB Postgres Advanced Server](/epas/latest) or [PostgresSQL](/supported-open-source/postgresql/) version 11 or later to hold the Postgres Enterprise Manager backend database. 
   
-  1. Install the Postgres server. See [Installing EDB Postgres Advanced Server on Linux](/epas/latest/epas_inst_linux) or [Installing PostgreSQL](/supported-open-source/postgresql/installer/).
+  1. Set up the repository
+
+     Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
+
+     -  To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+
+     -  To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
```

Comment:
We could add a note here that this is only required if you are not using an EPAS backend (and even then you can sometimes do without it as Jun found!)

Reply:
added note and committed